tensorflow_encrypted/protocol/pond.py

```diff
@@ -818,9 +821,12 @@ def add(self, other):
     def __add__(self, other):
         return self.prot.add(self, other)
 
-    def sum(self, axis, keepdims=False):
+    def sum(self, axis=0, keepdims=False):
```

Comment:
if TF only has `reduce_sum` then maybe we should simply follow that to keep it simple?

Reply:
depends if you want to be friendly with numpy users?

Reply:
wdyt?

Reply:
could we swap so that `reduce_sum` is called by `sum` at least? (okay this is vanity, I know..)

Reply:
I don't mind having aliases, but it needs to be mentioned in the doc so people don't think it's a different behavior under the hood.

Reply:
that was my concern as well. my vote is on sticking with TF (since we're promoting that anyway) but okay with catering a bit for numpy users as well if you guys think it's worth it